extensions/amp-accordion/1.0/base-element.js

```diff
@@ -219,7 +215,7 @@ function ContentShimWithRef(
   const contentRef = useRef();
   contentRef.current = contentElement;
   useSlotContext(contentRef);
-  useImperativeHandle(ref, () => contentElement, [contentElement]);
+  ref.current = contentElement;
```

Comment:
I'm not sure this is a good idea. The `forwardRef` API for `ref` is `{current: T} | function(T)`. Thus you'd have to support both `ref.current = X` and `ref(X)` forms. The `useImperativeHandle` does that for you.